refactor: use trafilatura.fetch_url over httpx.get

Amolith opened the pull request:
`httpx.get` was failing to fetch the contents of a site, but `trafilatura.fetch_url` worked fine, so I swapped importing all of `httpx` just for the `get` function with importing an additional function from the library it already uses. No worries if you want to keep `httpx`, though, this just resolved the issue for me and I thought I'd bump it upstream.

daturkel commented:
Huh I have no idea why that would be the case. Do you have an example url?

I suspect this will break the tests and I'll need to remove httpx as a requirement. I'm on vacation so cant do this now but will try to look when i return

Amolith commented:
It was a local news site, I'll look around and see if I can find a less sensitive example after taking a stab at the tests.

Amolith committed:
refactor(fetcher): switch to trafilatura for url fetching

Removed dependency on `httpx` by utilizing `trafilatura.fetch_url`
which is already part of the project's dependencies.

schneiderfelipe commented:
Sorry if the question is dumb, but is this related to the following error?

```console
$ llm -f site:https://llm.datasette.io/ -t fabric:summarize
Error: Could not load fragment site:https://llm.datasette.io/: No text found
```
